metron-interface/metron-rest/src/test/java/org/apache/metron/rest/config/TestConfig.java

```diff
@@ -205,8 +218,10 @@ public RestTemplate restTemplate(StormCLIWrapper stormCLIClientWrapper) {
   }
 
   @Bean
-  public AdminUtils$ adminUtils() {
-    return AdminUtils$.MODULE$;
+  public AdminClient adminUtils(KafkaComponent kafkaWithZKComponent) {
+    Map<String, Object> adminConfig = new HashMap<>();
+    adminConfig.put("bootstrap.servers", kafkaWithZKComponent.getBrokerList());
+    return KafkaAdminClient.create(adminConfig);
```

Comment:
Just refer to `AdminClient` instead.  Explanation provided in the other comment.
```
return AdminClient.create(adminConfig);
```

Reply:
Done